spec/controllers/profiles_controller_spec.rb

```diff
@@ -62,7 +62,7 @@
       it "should soft delete users playlists" do
         expect do
           put :update, params: { user_id: 'sudara', profile: { bio: 'very spammy bio' } }
-        end.to change(Comment, :count).by(-10)
+        end.to change(Comment, :count).by(-9)
```

Comment:
This is now also part of https://github.com/sudara/alonetone/pull/526
Will go away once we merge that PR